.github/policies/resourceManagement.yml

```diff
@@ -0,0 +1,1882 @@
+    - description: Add no-recent-activity label to PRs
+      frequencies:
+      - hourly:
+          hour: 6
+      filters:
+      - isPullRequest
+      - isOpen
+      - hasLabel:
+          label: needs-author-action
+      - noActivitySince:
+          days: 14
+      - isNotLabeledWith:
+          label: no-recent-activity
+      actions:
+      - addLabel:
+          label: no-recent-activity
+      - addReply:
+          reply: This pull request has been automatically marked `no-recent-activity` because it has not had any activity for 14 days. It will be closed if no further activity occurs within 14 more days. Any new comment (by anyone, not necessarily the author) will remove `no-recent-activity`.
+    - description: Close issues with no recent activity
+      frequencies:
+      - hourly:
+          hour: 6
+      filters:
+      - isIssue
+      - isOpen
+      - hasLabel:
+          label: no-recent-activity
+      - noActivitySince:
+          days: 14
+      actions:
+      - addReply:
+          reply: This issue will now be closed since it had been marked `no-recent-activity` but received no further activity in the past 14 days. It is still possible to reopen or comment on the issue, but please note that the issue will be locked if it remains inactive for another 30 days.
+      - closeIssue
+    - description: Close PRs with no-recent-activity
+      frequencies:
+      - hourly:
+          hour: 6
+      filters:
+      - isPullRequest
+      - isOpen
+      - hasLabel:
+          label: no-recent-activity
+      - noActivitySince:
+          days: 14
+      actions:
+      - addReply:
+          reply: This pull request will now be closed since it had been marked `no-recent-activity` but received no further activity in the past 14 days. It is still possible to reopen or comment on the pull request, but please note that it will be locked if it remains inactive for another 30 days.
+      - closeIssue
+    - description: Close inactive Draft PRs
+      frequencies:
+      - hourly:
+          hour: 6
+      filters:
+      - isDraftPullRequest
+      - isOpen
+      - noActivitySince:
+          days: 30
+      actions:
+      - closeIssue
+      - addReply:
+          reply: Draft Pull Request was automatically closed for 30 days of inactivity. Please [let us know](https://github.com/dotnet/runtime/blob/main/docs/area-owners.md) if you'd like to reopen it.
+    eventResponderTasks:
+    - if:
+      - or:
+        - payloadType: Issues
+        - payloadType: Pull_Request
+      - isAction:
+          action: Opened
+      then:
+      - if:
+        - hasLabel:
+            label: area-AssemblyLoader-coreclr
+        then:
+        - mentionUsers:
+            mentionees:
+            - vitek-karas
+            - agocke
+            - vsadov
+            replyTemplate: >-

```

Comment:
It's a shame the new config format has to repeat this `replyTemplate` for every label that triggers a mention, causing a lot of duplication, but oh well. We might find a way to de-dupe this later (or just move our label mentions to a GitHub Action at some point).